### PR TITLE
MPP-2995 Push footer to bottom of desktop screens

### PR DESCRIPTION
**Pull request by codemist**
Quick fix for issue #3223 and MPP-2995. This sets a media query to set the body `width` to `100%` for large device widths (targeting desktop users), without triggering the bug found on mobile browsers related to MPP-2920.

### CODE REVIEW

File: frontend/src/styles/globals.scss [hunk 1]
```diff
@@ -28,6 +28,11 @@ body,
 #__next,
 #overlayProvider {
   font-family: $font-stack-base;
+
+  @media screen and #{$mq-lg} {
+    // Ensures that the footer stays at the bottom
+    height: 100vh;
```
Comment (maxxcrawford) [inline frontend/src/styles/globals.scss:34]: Question: What does this do when there's more content than `100vh`?  Should this be `min-height` instead? Does that have the same effect? 

Also, would doing a [screen _height_ media query](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/height) be helpful?

**Review by maxxcrawford — COMMENTED** [review]
Added question/comment

File: frontend/src/styles/globals.scss (hunk 1 above)
Comment (Vinnl) [reply frontend/src/styles/globals.scss:34]: Yeah, IIRC the original was `100%`, which should allow it to grow to the full height of its parent. (Specifically, the footer is in an element with `display: flex; flex-direction: vertical`, and the element above the footer has a `flex-grow` higher than 0. That allows it to push the footer to the bottom, as long as all its parents elements have `height: 100%`.)

Which is not to say that `100vh` might not work or that you may not have good reasons for using that - just sharing the context behind the original value :)

File: frontend/src/styles/globals.scss [hunk 2]
```diff
@@ -28,6 +28,11 @@ body,
 #__next,
 #overlayProvider {
   font-family: $font-stack-base;
+
+  @media screen and #{$mq-lg} {
+    // Ensures that the footer stays at the bottom
+    height: 100%;
```
[the remark this answers is not in the document]
Comment (Vinnl) [reply frontend/src/styles/globals.scss:34]: Looks good! Maybe add a comment with a link to the bug/previous PR to tell future readers why this is in a media query?

**Review by Vinnl — APPROVED** [review]
Works for me!

**Pull request merged**